docs/metrics.md

```diff
@@ -85,18 +37,18 @@ We will aim to collect data that will help us understand the following (please a
 - Where are the drop-off points in the user flow?
     - Do the majority of people make it all the way through the setup process?
     - Once initially setup, do people continue to add credentials?
-    - Do people continue to auto-fill credentials after the first usage?
 - What are people's opinions on LB
     - Feedback forms
 - Do people sync their passwords **(not in alpha)**?
     - How does syncing affect engagement?
-- Maybe: what type of sites do people use Lockbox with **(not in alpha)**?
+- What type of sites do people use Lockbox with?
+    - Do they use Lockbox only for their most sensitive accounts?
 
 ## Collection
 
 **Note:** *This is the collection plan for our internal alpha release. For our beta release will be taking advantage of test pilot's telemetry API. The metrics plan for beta will be described in a separate document.*
 
-At this point, all measurements related to Lockbox will be made client-side. All users will have to authenticate through FxA, and thus additional measurements related to that will be logged on the FxA auth server. We have no control over what is already collected via that mechanism.
+At this point, all measurements related to Lockbox will be made client-side. Eventually all users will have to authenticate through FxA, and thus additional measurements related to that will be logged on the FxA auth server. We have no control over what is already collected via that mechanism.
```

Comment:
Minor point, but I thought our goal was to turn FxA into an option that we let users upgrade to later.

Reply:
>I thought our goal was to turn FxA into an option that we let users upgrade to later.


@jimporter Good point, it is not expected to be always required.

@irrationalagent Based on our discussion this morning, do you want to take a pass at re-wording this? I can if you're busy.